.github/workflows/on_release_publish.yml

```diff
@@ -18,7 +19,7 @@ jobs:
 
     # when it's not a release build use the commit hash for the version tag
     - name: Set tegola version (use commit hash)
-      if: github.event_name == 'push'
+      if: github.event_name != 'release'
       run: echo ${{ github.sha }} | cut -c1-7 > ${{ github.workspace }}/version.txt
```

Comment:
I wonder if we can replace this with a git attribute filter:
https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes#_keyword_expansion